bootstrap.ps1

```diff
@@ -79,3 +89,17 @@ Pop-Location
 Push-Location (Join-Path $PSScriptRoot 'src/QsCompiler/QirGeneration')
 .\FindNuspecReferences.ps1;
 Pop-Location
+
+try {
+    Push-Location (Join-Path $PSScriptRoot "src" "QirTools" "pyqir")
```

Comment:
Nice to have this wrapped in a try/finally so that exceptions don't corrupt the location stack. I keep meaning to write an `Invoke-AtLocation` cmdlet that takes a scriptblock and a path to do this...

```pwsh
Invoke-AtLocation (Join-Path $PSScriptRoot "src" "QirTools" "pyqir") {
     ...
}
```

Reply:
We have `exec` now that I added to `utils.ps1` and use ` -ErrorAction SilentlyContinue` if you don't want to fail if the command fails.